socialhome/users/tests/test_views.py

```diff
@@ -146,6 +148,18 @@ def test_contents_queryset_returns_content_in_correct_order(self, admin_client,
         assert qs[1].id == contents[1].id
         assert qs[2].id == contents[0].id
 
+    def test_content_saved_in_correct_order(self, admin_client, rf):
+        request, view, contents, profile = self._get_request_view_and_content(rf, create_content=False)
+        
+        pinned_content_1 = Content(pinned=True, text="foobar")
+        pinned_content_2 = Content(pinned=True, text="foobar")
+        pinned_content_3 = Content(pinned=True, text="foobar")
+        pinned_content_1.save(author=profile)
+        pinned_content_2.save(author=profile)
+        pinned_content_3.save(author=profile)
+
+        assert {pinned_content_1.order, pinned_content_2.order, pinned_content_3.order} == {1, 2, 3}
```

Comment:
A Python set is not ordered, so this guarantees only that the elements are in the set: https://docs.python.org/3/library/stdtypes.html#set-types-set-frozenset